analyzers/tests/SonarAnalyzer.UnitTest/TestCases/AsyncVoidMethod.cs

```diff
@@ -141,4 +141,17 @@ public void SomeMethod()
             await Task.Run(() => Console.WriteLine("test"));
         }
     }
+
+    // https://github.com/SonarSource/sonar-dotnet/issues/5432
+    public class Reproducer5432
+    {
+        public void SomeMethod()
+        {
+            var _timer = new System.Threading.Timer(RunOnceAsync);
+        }
+
+        private async void RunOnceAsync(object _) // Noncompliant FP
```

Comment:
To make it easier to understand in the future. It's clear now, as now we have the context.  And the case is quite special.
```suggestion
        private async void RunOnceAsync(object _) // Noncompliant FP, cannot change signature because it's used as a delegate
```